Gemfile.rails4

```diff
@@ -7,3 +7,4 @@ gemspec
 gem 'rails', '~>4'
 # this is used for testing gem tracking
 gem 'rainbow', require: false
+gem 'pundit'
```

Comment:
Needed to add another gem since rainbow is require false here. Not sure if it needs to be require: false?

Reply:
I think I did that on purpose on rainbow but we could likely revisit it as a lot has changed out how we load initial Coverage since then